OSDOCS#11414: IPI improvements - Part 2 (#79537)
openshift/openshift-docs: modules/installation-azure-confidential-vms.adoc

.Procedure

* Edit the `install-config.yaml` file before deploying your cluster:

> ** To enable confidential VMs only on control plane, add the following stanza:

Reviewer: ```suggestion
** Enable confidential VMs only on the control plane by adding the following stanza:
```

Reviewer: I prefer the direct syntax because I am thinking of someone searching the web for "enabling confidential VMs on..."

+
[source,yaml]
----
controlPlane:
  platform:
    azure:
      settings:
        securityType: ConfidentialVM
        confidentialVM:
          uefiSettings:
            secureBoot: Enabled
            virtualizedTrustedPlatformModule: Enabled
      osDisk:
        securityProfile:
          securityEncryptionType: VMGuestStateOnly
----

> **  To enable confidential VMs only on compute nodes, add the following stanza:

Reviewer: ```suggestion
**  Enable confidential VMs only on compute nodes by adding the following stanza:
```

            virtualizedTrustedPlatformModule: Enabled
      osDisk:
        securityProfile:
          securityEncryptionType: VMGuestStateOnly
----

> **  To enable confidential VMs on all nodes, add the following stanza:

Reviewer: ```suggestion
**  Enable confidential VMs on all nodes by adding the following stanza:
```
